src/node_funcs.c

```diff
@@ -1,6 +1,5 @@
 #include "postgres.h"
 
-#include "catalog/pg_type.h"
```

Comment:
We use `INT4OID` in this file. Does this mean that `pg_type_d` is included indirectly somehow?

Reply:
I hate the fact that includes are so fuzzy. I try to follow this principle: If the current file references something from a header, that header should be a direct include and not transitive. In this case I'm not sure. `pg_type_d` is not public because it's autogenerated, and personally I use `pg_type` when I reference core type OIDs.

Reply:
Let's then keep things as they are then, I had no more finds here